### ISSUE TITLE
Midpoint Review [DGIB, Secure Inbox/Messaging (for education)]

### ISSUE THREAD

**Issue by rlyons89** [department-of-veterans-affairs/va.gov-team#30622]
# Steps to complete Midpoint Review 

## Before meeting

**VFS Product Manager**
- [X]  Create this issue and notify the Collaboration Cycle team of this ticket in the in #vfs-platform-support ([see Slack Workflow (User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)). 
- [ ]  Add artifacts listed below to the ticket at least 2 days before scheduled meeting. 

**Platform**
- [ ] Schedule meeting with Collaboration Cycle reviewers and VFS team attendees listed in the ticket. 
- [ ] Create feedback tickets and link to the Midpoint Review epic (if no feedback, please comment stating that). 

## After meeting
**VFS Product Manager**
- [ ] Respond to feedback tickets.
- [ ] Close this ticket when review is complete and feedback has been addressed.  

**Platform**
- [ ] Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061). 

### Has your team met with the Sitewide Content team for content support? 
- [ ] Yes*
- [ ] No

\*If yes, please include any relevant content artifacts.

## Artifacts

Provide links or documents for the following: 

- [ ] Product outline: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/product-outline-template.md
- [ ] Finalized design prototype or mockup
- [ ] Research plan 

 
- [ ] Conversation guide 

### Content artifacts 

- [ ] Content brief (if applicable) and your product’s content source of truth 

- [name of content of truth, for example, "Invision Prototypes"](link here)

- [ ] Github ticket for any relevant static content page and entry point updates for tool or feature (if applicable)
 
### QA artifacts

[Learn more about QA artifacts](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)

- [ ] Regression test plans (if there isn't already a regression test plan for this feature in TestRaiProduct)
- [ ] Link to test cases/test plan in TestRail (draft OK) 

### Required* 
\*Only required for synchronous meetings.
- Product Manager: Shamik Desai
- OCTO-DE Product lead: Matt Self
- OCTO-DE Design lead: Ryan Thurlwell

### Team Members

- Russell Lyons
- Leelah Holmes
- Alexander Ferzola
- Audra Ayotte

### VFS Attendees

- Content and IA: 
- Accessibility: 
- QA specialists: 

## Scheduling
Would you like this review to be asynchronous? 

Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review.
- [X] Yes
- [ ] No

**[Tue, October 5]** **[11:30am – 12:00pm]**

## Additional information
Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Midpoint Review Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) on Platform website for more information about the Collaboration Cycle.

**Comment by peggygannon** [comment]
Hello @rlyons89 Midpoint Review has been scheduled for Oct 5, at 11:30 a.m. 

Please be sure to upload artifacts at least 2 days before this meeting. Thanks!

**Comment by peggygannon** [comment]
Hi, @rlyons89 Checking in on the artifacts for next Tuesday's midpoint review. Can you upload a prototype and any other necessary artifacts for reviewers? Thanks!

**Comment by rlyons89** [comment]
@peggygannon yes, just added some of the updated mockups and user flows to the ticket

**Comment by pjhill** [comment]
@rlyons89 , Sorry for the last minute request -- are there any artifacts related to QA activities on this effort?

**Comment by rlyons89** [comment]
@pjhill no QA artifacts yet, we do not have a dev timeline set for this feature yet so we haven't spent time on those materials. We will look to establish a dev timeline after we conduct usability testing and address our findings.

**Comment by jimuhh** [comment]
So as I understand it, the problem statement-style description of issues prompting redesign (like the one found in product outline) is awaiting usability testing results?

**Comment by pjhill** [comment]
> @pjhill no QA artifacts yet, we do not have a dev timeline set for this feature yet so we haven't spent time on those materials. We will look to establish a dev timeline after we conduct usability testing and address our findings.

Thanks for the quick response! Given that there are no QA artifacts to review at this time, Platform QA has no feedback.

**Comment by rlyons89** [comment]
IA request ticket mentioned during midpoint review meeting: https://github.com/department-of-veterans-affairs/va.gov-team/issues/30790#issuecomment-934503627
